Tasks/Rabushenka/Tasks5/HomWr-5.py

```diff
@@ -0,0 +1,54 @@
+# Еxercise 1
+from math import sqrt
+question = input("Input: ")
+def string_symbols(string):
+    dictionary = {"Upper case": 0, "Lower case": 0}
+    for symbols in string:
+        if symbols.isupper():
+           dictionary["Upper case"]+=1
+        elif symbols.islower():
+           dictionary["Lower case"]+=1
+        else:
+           continue
```

Comment:
a simple else would suffice, all symbols which are not in upper case could be considered lower